.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This is the app's ESLint configuration file. These things can get a little crazy, so I left it simple.